Dockerfile

```diff
@@ -1,12 +1,19 @@
+FROM alpine:3.18
+
+WORKDIR /root
+
+RUN mkdir -p /usr/share/GeoIP
+COPY ./GeoLite2-City.mmdb /usr/share/GeoIP/GeoLite2-City.mmdb
+COPY ./GeoLite2-Country.mmdb /usr/share/GeoIP/GeoLite2-Country.mmdb
+
 # Dockerfile - alpine
 # https://github.com/openresty/docker-openresty
 
 ARG RESTY_IMAGE_BASE="alpine"
 ARG RESTY_IMAGE_TAG="3.18"
 
-FROM ${RESTY_IMAGE_BASE}:${RESTY_IMAGE_TAG}
 
-LABEL maintainer="DQDevops"
+LABEL maintainer="Evan Wies <evan@neomantra.net>"
 
 # Docker Build Arguments
 ARG RESTY_IMAGE_BASE="alpine"
```

Comment:
Aren't we migrating from OpenSSL 1.1.f to 3?
If so, shouldn't these ARGs be updated?
`ARG RESTY_OPENSSL_VERSION="1.1.1u"`
`ARG RESTY_OPENSSL_PATCH_VERSION="1.1.1f"`